### DISCUSSION TITLE
Hot restart issue on iPhone running iOS 14.4

### DISCUSSION THREAD

**Discussion by GJKuz** [xamarin/Xamarin.Forms#13644] [Q&A] [answered]
Hi, I am using VS 2019 community edition 16.8.4 on my SP3 running W10 20H2 with all the latest and greatest patches and updates available for the device. I created a new Xamarin Forms mobile project to test the hot restart on my iPhone 12 pro max running iOS 14.4. I setup everything correctly under iOS Project Settings -> iOS Bundle Signing. I also have iTunes installed. My phone appears in the VS project, and I have no issues running it in debug mode. From what I can see from the output window, there were no issues with the deployment. Even got the prompt asking to launch the app on my device to continue. But I have two issues.

1) Any Xamarin forms project deployed through VS2019 won't appear on my iPhone unless I reboot my device (12 pro max) after each deployment.
2) Once I launch the app, the splash screen appears and then the app crashes.

I tried this on my desktop computer running W10 as well. The exact same behavior. App1 is just a blank Xamarin Forms mobile app. This is my first time using Xamarin Forms. 

I don’t have a MAC device, so I was very excited about the hot restart feature. Any idea what could be going on? Anything I could do to investigate this further? Any help is greatly appreciated. Thanks.

**Comment by ebrodlic** [comment, marked as answer]
I have the exact same problem when it comes to crashing right after splash screen! I've made an issue about it so let's hope it gets resolved: #13759

**Reply by bernmuel** [reply]
I have the same problem with iOS 14.4.1.  Its crashing right after the splash screen

**Reply by ebrodlic** [reply]
The VS 16.9.2 update fixes it, so give it a try.

**Reply by bernmuel** [reply]
The VS 16.9.2 update works for me

**Reply by GJKuz** [reply]
Thank you both for taking the time to comment. @ebrodlic Thank you. I had trouble with my test project even after the updating to 16.9.2. But I was able to get a brand new project to work after an iPhone restart. I will take that. :)
